.github/workflows/main.yml

```diff
@@ -18,7 +18,8 @@ jobs:
         LISP_IMPL: ["sbcl", "allegro", "ccl"]
         COALTON_ENV: ["release", "development"]
         SAFETY: ["0", "3"]
-        DISABLE_SPECIALIZATION: ["1", "0"] 
+        DISABLE_SPECIALIZATION: ["1", "0"]
+        HEURISTIC_INLINING: ["1", "0"]
```

Comment:
Is it fine with double the tests, with heuristic inlining enabled or disabled like this?

Reply:
Yes.